tests/transformers/tests/generation/test_utils.py

```diff
@@ -3121,15 +3209,15 @@ def forward(self, input_ids, **kwargs):
     def test_default_max_length_warning(self):
         model = AutoModelForCausalLM.from_pretrained("hf-internal-testing/tiny-random-gpt2").to(torch_device)
         tokenizer = AutoTokenizer.from_pretrained("hf-internal-testing/tiny-random-gpt2")
-        model.config.pad_token_id = tokenizer.eos_token_id
+        model.generation_config.pad_token_id = tokenizer.eos_token_id
 
         text = "Hello world"
         tokenized_inputs = tokenizer([text], return_tensors="pt")
         input_ids = tokenized_inputs.input_ids.to(torch_device)
 
-        # Default generation config value of 20 -> emits warning
-        with self.assertWarns(UserWarning):
-            model.generate(input_ids)
+        # # Default generation config value of 20 -> emits warning
```

Comment:
Can you comment why did you comment and disable this part?

Reply:
https://github.com/malkomes/optimum-habana/blob/658decbc459e2493b1d8779fdf8d79b73eb2c2d6/tests/transformers/tests/generation/test_utils.py#L3163C9-L3163C50